lib/rest/api/v2010/account/address.js

```diff
@@ -504,7 +499,8 @@ AddressPage.prototype.getInstance = function getInstance(payload) {
  *
  * @param {Twilio.Api.V2010} version - Version of the resource
  * @param {object} payload - The instance payload
- * @param {sid} accountSid - The account_sid
+ * @param {sid} accountSid -
```

Comment:
Ideally these types such as 'sid' should be typed as `string` in JSDoc comments. Same goes for the other ones. It's cool if we create custom type aliases like `type Sid = string` for them in TypeScript (that's what I did with some of them) but I think for the JSDoc comments actually naming them as strings and then having in the description that they have to have the shape of a SID (or a specific date style) seems more reasonable

Reply:
Since the type is already `sid` and this change is so massive, I would rather leave fixing this for another time.